src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,17 @@
+package lotto.domain;
+
+public class Money {
+
+    private final int value;
+
+    public Money(final int value) {
+        this.value = value;
+    }
+
+    public int Value() {
```

Comment:
객체의 상태 값을 외부로 꺼낼 때는 아래와 같이 `getter` 메서드를 생성해주세요! 😃
```suggestion
    public int getValue() {
```